.github/workflows/lint.yml

```diff
@@ -1,12 +1,22 @@
 on: pull_request
-name: Lint check
+name: Lint checks
```

Comment:
Hey, @NimishVerma I believe this file `.github/workflows/lint.yml` should not be changed. Probably a problem merging master into the branch. I understand this PR should only change the `.pre-commit-config.yaml` file.